crates/core/src/lib.rs

```diff
@@ -42,44 +42,30 @@
 //!
 //! You can find more examples in the [`examples`](https://github.com/penberg/libsql-experimental/tree/libsql-api/crates/core/examples) directory.
 
-pub mod connection;
-pub mod database;
-pub mod errors;
-pub mod params;
-pub mod rows;
-pub mod statement;
-pub mod transaction;
+// Legacy mode, for compatibility with the old libsql API.
+pub mod v1;
 pub mod v2;
 
-pub type Result<T> = std::result::Result<T, errors::Error>;
-type BoxError = Box<dyn std::error::Error + Send + Sync>;
+pub use v1::{
+    database::Opts,
```

Comment:
This was originally behind a feature flag for replication? I wanted to revisit how we did feature flags anyways so we can merge this for now and get this fixed in a follow up.